.goreleaser.yml

```diff
@@ -1,23 +1,25 @@
 project_name: influxdb
 builds:
-  - goos:
```

Comment:
There are a lot of whitespace changes in here. Did you format it with a particular tool? This just makes the diff a little hard to read, so I'm curious.

Reply:
VSCode. Can unformat the file if preferred